komodo/lint_maturity.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+import yaml
+from packaging.version import Version, parse
+
+
+INVALID_TAGS = {
+    "a": ["Invalid"],
+    "b": ["a", "Invalid"],
+    "rc": ["a", "b", "Invalid"],
+    "None": ["a", "b", "rc", "Invalid"],
+}
+
+
+def get_parser():
+    parser = argparse.ArgumentParser(description=("Lint the maturity of packages."))
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument(
+        "--release_file",
+        type=lambda arg: arg
+        if os.path.isfile(arg)
+        else parser.error("{} is not a valid file".format(arg)),
+        help="Release file",
+    )
+    group.add_argument(
+        "--release_folder",
+        type=lambda arg: arg
+        if os.path.isdir(arg)
+        else parser.error("{} is not a valid directory".format(arg)),
+        help="File with all package tags named as release version.",
+    )
+    return parser
+
+
+def define_release_version(release_file):
+    try:
+        Version(release_file)
+        release_version = (
+            "None"
+            if parse(release_file).pre is None
+            else str(parse(release_file).pre[0])
+        )
+    except:
```

Comment:
Blanket `except` is usually not a good idea, you want to catch specific errors, otherwise it can lead to some quite strange behavior.

Reply:
In the end this exception was not required, but I learned that I need to import the error from the package. Many thanks.